### DISCUSSION TITLE
It's shrunk!

### DISCUSSION THREAD

**Discussion by rszemeti** [peterantypas/maiana#36] [Show and tell]
I didn't reallise just how much you had shrunk it until the boards turned up this morning!  Small enough to make a PLB with this!
![20210108_153718](https://user-images.githubusercontent.com/46870724/104036106-e8729880-51ca-11eb-8e19-7252e541df20.jpg)

**Comment by peterantypas** [comment]
OMG ... that board ... I don't have any of those laying around anymore :D

Yeah, I suppose a PLB is also a possibility. It did come up when I was exploring regulatory approval, in fact the lab thought that this was the application given the design and size. I had never thought of that before, I just wanted to build a waterproof VHF antenna with a built-in transponder for my boat ...

**Comment by rszemeti** [comment]
Yep, it worked, I had some issues with loading the software as the original
dump of code didn't include an upload script, and I had significant issues
even then, which took me a while to debug but basically came down to clock
configuration ... I could upload a "blinky" program and it would run fine,
but the AIS code crashed and burned during bootup. It took me ages to find
the cause, which was clock configuration ... as it switched from the
internal to the external clock, the multipliers or something were wrong and
it crapped itself.   I got it in the end, but i was considerably greyer by
then ;)
